### PR TITLE
Feature/licences page

### PR DESCRIPTION
**Pull request by andrewhick**
Ready for review, all tests pass including Rake CI

### CODE REVIEW

File: features/page_objects/licences_page.rb [hunk 1]
```diff
@@ -3,11 +3,31 @@ class LicencesPage < SitePrism::Page
   # Your water abstraction licences
 
   element(:changepw, ".header-links a:nth-child(1)")
-  elements(:licences, ".license-result:nth-child(128) .license-result__column")
+  # elements(:licences, ".license-result:nth-child(128) .license-result__column")
```
Comment (timstone) [inline features/page_objects/licences_page.rb]: Removed commented out lines unless they are for information

File: features/step_definitions/search_filter_steps.rb [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  @expected_search_result = "28/09"
+  @expected_result_count = 4
+  @front_app.licences_page.search(
+    search_form: @expected_search_result.to_s
+  )
+end
+
+Given(/^all licences containing that term are shown on screen$/) do
+  @licences_page = @front_app.licences_page
+  puts "Licences count is:   " + @front_app.licences_page.view_links.count.to_s
```
Comment (timstone) [inline features/step_definitions/search_filter_steps.rb]: Again remove the puts

File: features/step_definitions/search_filter_steps.rb [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+Given(/^I remove a search term$/) do
+  @expected_search_result = ""
+  @expected_result_count = @total_licences
+  @front_app.licences_page.search(
+    search_form: @expected_search_result.to_s
+  )
+end
+
+Given(/^I can see the original number of licences$/) do
+  puts "Licences count is: " + @front_app.licences_page.view_links.count.to_s
```
Comment (timstone) [inline features/step_definitions/search_filter_steps.rb]: Puts puts puts puts

File: features/step_definitions/search_filter_steps.rb [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  expect(@front_app.licences_page.licencename_link.text).to have_text("Licence name ▲")
+  puts "First licence: " + @front_app.licences_page.firstlicence.text.to_s
+  puts "Last licence:  " + @front_app.licences_page.lastlicence.text.to_s
+end
+
+Given(/^I select the licence number heading$/) do
+  @front_app.licences_page.serialno_link.click
+end
+
+Given(/^the table is sorted by licence number in ascending order$/) do
```
Comment (timstone) [inline features/step_definitions/search_filter_steps.rb]: You could probably do something with having an array of elements sorting them in order and then asserting the first shown on the page is the same as the first element in the array. What you've done is ok but is hard coded to the test data so more brittle. I'm sure Alan can advise if you wan't to look into options.

Comment (andrewhick) [reply features/step_definitions/search_filter_steps.rb]: Will try and work this out for the next iteration.

**Pull request merged**